src/ui/suspense/frameStepsCache.ts

```diff
@@ -1,41 +1,19 @@
 import { FrameId, PauseId, PointDescription } from "@replayio/protocol";
 
+import { createUseGetValue } from "bvaughn-architecture-demo/src/suspense/createGenericCache";
 import {
-  createGenericCache,
-  createUseGetValue,
-} from "bvaughn-architecture-demo/src/suspense/createGenericCache";
-import { Pause } from "protocol/thread/pause";
-import { assert } from "protocol/utils";
-
-export const {
-  getValueSuspense: getFrameStepsSuspense,
-  getValueAsync: getFrameStepsAsync,
-  getValueIfCached: getFrameStepsIfCached,
-} = createGenericCache<[pauseId: PauseId, frameId: FrameId], PointDescription[] | undefined>(
-  async (pauseId, frameId) => {
-    const pause = Pause.getById(pauseId);
-    assert(pause, `no pause for ${pauseId}`);
-
-    // Handle potential errors by swallowing and returning `undefined`
-    // TODO [BAC-2459] This may not be necessary if the backend
-    // stops throwing internal errors
-    try {
-      const steps = await pause.getFrameSteps(frameId);
-      return steps;
-    } catch (err) {
-      return undefined;
-    }
-  },
-  (pauseId, frameId) => `${pauseId}:${frameId}`
-);
+  getFrameStepsAsync,
+  getFrameStepsIfCached,
+} from "bvaughn-architecture-demo/src/suspense/FrameStepsCache";
+import { ReplayClientInterface } from "shared/client/types";
 
 export const useGetFrameSteps = createUseGetValue<
```

Comment:
I think I'm getting a little less clear on the purpose of this cache (in `src/ui/suspense`) as being separate from the caches in bvaughn